User management update (#1138)
rudderlabs/rudderstack-docs: docs/dashboard-guides/user-management.mdx

---
title: "Members (User Management)"

> description: Invite team members and create roles for them in RudderStack.

Reviewer: ```suggestion
description: Invite team members and assign relevant roles to them in RudderStack.
```

---

RudderStack's user management feature lets you easily collaborate with other members of your organization. It provides different options to manage users and their permissions in your RudderStack workspace. 

## Member limit

The following table specifies the number of team members you can add to your workspace in different [RudderStack Cloud plans](https://www.rudderstack.com/pricing/):

> | Plan | Member limit |

Reviewer 2: Should we add a link to pricing page instead of duplicating that information? Otherwise they it is hard to keep them in sync

Author: we've already added it above in L10. We want to explicitly highlight this limit in this guide as well.

| :-----| :------|

> | [Free](https://app.rudderstack.com/signup?type=freetrial) | 10 |

Reviewer 2: `?type=freetrial` is not needed now. By default user lands on free trial signup

Author: got it - made the change!

> | [Free](https://app.rudderstack.com/signup?type=freetrial) | 10 |

Author: ```suggestion
| [Free](https://app.rudderstack.com/signup) | 10 |
```
